webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -24,3 +28,39 @@ export const addSearchToURL = (path, query) => {
   url.addSearch({ ...query, include_permissions: true });
   return url.toString();
 };
+
+export const getDuration = (start, finish) => {
+  if (!start && !finish)
+    return { text: __('N/A'), tooltip: __('No start or end dates') };
+
+  if (!start && finish) {
+    return { text: __('N/A'), tooltip: __('Task was canceled') };
+  }
+
+  const dateOptions = {
+    largest: 1,
+    language: documentLocale(),
+    fallbacks: ['en'],
+    round: true,
+  };
+
+  start = new Date(isoCompatibleDate(start));
+  if (start && !finish) {
+    finish = new Date();
+    const duration = finish - start;
+    return {
+      text: `${humanizeDuration(duration, dateOptions)} +`,
+      tooltip: `${__('More than')} ${numberWithDelimiter(
+        (finish - start) / 1000
```

Comment:
you can use the duration const which is declared above, instead of re-calculating `finish - start`

Reply:
Good catch, thanks!